### PR TITLE
zynqmp: initial manifest

### PR DESCRIPTION
**Pull request by ierkiaga**
Initial manifest document to build ZynqMP platform. In support to PR522
for build repository.

Signed-off-by: Ibai Erkiaga <ibai.erkiaga-elorza@xilinx.com>

### CODE REVIEW

**Review by jforissier — APPROVED** [review]
`Acked-by: Jerome Forissier <jerome@forissier.org>`

**Comment by ierkiaga** [comment]
Hi @jforissier 
Should I change the URL and branch of the build repository to upstream build repo in the final commit?

**Comment by jforissier** [comment]
> Hi @jforissier
> Should I change the URL and branch of the build repository to upstream build repo in the final commit?

Absolutely, and I should have noticed :wink:

File: zynqmp.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	<remote name="github"   fetch="https://github.com" />
+
+	<default remote="github" revision="master" />
+
+	<!-- OP-TEE gits -->
+	<project path="optee_client"	name="OP-TEE/optee_client.git" />
+	<project path="optee_docs"	name="OP-TEE/optee_docs.git" />
+	<project path="optee_os"	name="OP-TEE/optee_os.git"/>
+	<project path="optee_test"	name="OP-TEE/optee_test.git" />
+	<project path="build"		name="ierkiaga/build"	revision="zynqmp" >
```
Comment (jforissier) [inline zynqmp.xml:12]: `name="OP-TEE/build.git"` and no revision

**Pull request merged**